src/averageAges.js

```diff
@@ -55,7 +70,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const arrDiffAge = (!onlyWithSon)
```

Comment:
I think better rename this way. Because this is array of children or only sons.
```suggestion
  const childrenWithMother = (!onlyWithSon)
```